lib/ansible/modules/monitoring/grafana_dashboard.py

```diff
@@ -100,6 +100,13 @@
     default: 'yes'
     type: bool
     version_added: 2.7
+  tls_insecure:
+    description:
+      - Enable insecure TLS protocols.
```

Comment:
Perhaps change this to `Enable insecure SSL protocols.`?
Since what we're enabling here is actually SSL, not TLS.
Or alternatively, explicitly mention that it's enabling SSLv2 and SSLv3?

Reply:
It also enables TLSv1 (if Python supports a newer TLS version) according to the PR description. So `Enable insecure TLS/SSL protocols.` would maybe be a better description?

Reply:
That'd work, yeah.